microsoft/testsuites/dpdk/dpdksuite.py

```diff
@@ -35,7 +35,7 @@
 
 VDEV_TYPE = "net_vdev_netvsc"
 MAX_RING_PING_LIMIT_NS = 200000
-DPDK_VF_REMOVAL_MAX_TEST_TIME = 60 * 10
+DPDK_VF_REMOVAL_MAX_TEST_TIME = 200
```

Comment:
What's the difference to change the timeout value? As my understanding, the longer timeout can prevent unstable status, unless the timeout is misused.

Reply:
I can change it back, it seems that more often the VF is added again but testpmd decides to ignore the kill signal and ends up running the entire time anyway. Maybe there's a better way to set the timeout where we can still get the stdout though.